nab/detectors/numenta/modelParams/model_params.json

```diff
@@ -61,7 +61,7 @@
             "seed": 1956,
             "spVerbosity": 0,
             "spatialImp": "cpp",
-            "synPermActiveInc": 0.002,
+            "synPermActiveInc": 0.003,
```

Comment:
@subutai can you wage on the importance (vs. specific-ness) of the changes? I'm just sorting the same problem in https://github.com/breznak/ML.benchmark/issues/14 , and I was assuming that the "state of the art" model params (for generic case) are in `examples/anomaly/temporal/hotgym`. Is it true? Will you propagate this change to nupic upstream?